spec/rails_helper.rb

```diff
@@ -73,5 +73,6 @@
     driven_by JS_DRIVER
   end
 
+  config.include FactoryBot::Syntax::Methods
```

Comment:
I've included same thing here https://github.com/vlado/earthquake-croatia/blob/b7c8c31847e9d7dcaa6b6487f475153aebe9b8c9/spec/rails_helper.rb#L70

Reply:
I guess if we add this in the same line git won't report it as merge conflict.

Reply:
Good catch, fixed.